backend/addcorpus/filters.py

```diff
@@ -31,6 +34,8 @@ class DateFilter(Filter):
     Filter for datetime values: produces two datepickers for min and max date.
     '''
 
+    mapping_types = ['date']
```

Comment:
```suggestion
    mapping_types = (MappingTypes.DATE,)
```
Same goes for rest of the mapping types. Tuple suggestion because this seems like immutable data (though single value tuple always looks a bit weird).